tools/pd-ctl/pdctl/command/store_command.go

```diff
@@ -706,7 +706,11 @@ func showAllStoresLimitCommandFunc(cmd *cobra.Command, args []string) {
 	cmd.Println(r)
 }
 
-func removeTombStoneCommandFunc(cmd *cobra.Command, _ []string) {
+func removeTombStoneCommandFunc(cmd *cobra.Command, args []string) {
```

Comment:
maybe `EnablePrefixMatching` is a dangerous operator for the user, do we need to disable it?

Reply:
ref https://github.com/tikv/pd/blob/d44d7212b3d3fce03adf0f8420bcd5c2cab7f7b3/tools/pd-ctl/pdctl/ctl.go#L32

Reply:
good idea